e2e_test.go

```diff
@@ -46,8 +46,12 @@ func (e *e2eTestRunner) run(name string, args ...string) []byte {
 
 	cmd := exec.CommandContext(e.ctx, name, args...)
 	cmd.Env = e.env
+	// Behaviour seems to be platform specific.
+	// Assure std in/err is not set otherwise combined output fails.
+	cmd.Stdin = nil
+	cmd.Stderr = nil
```

Comment:
This should not be possible for `cmd.Stdin` or `cmd.Stderr` to be non-nil, because this is the go code [here](https://cs.opensource.google/go/go/+/refs/tags/go1.18.2:src/os/exec/exec.go;l=169) which is not platform-specific. Also `cmd.CombinedOutput` would always fail for whatever platform is not setting these to `nil`, because of [this](https://cs.opensource.google/go/go/+/refs/tags/go1.18.2:src/os/exec/exec.go;l=558-563).

Please let me know how you detected this condition because that indicates a significant golang bug.

Reply:
Thank you for pointing this out; you are right, it certainly looks that way. 
Going to address this in a separate PR

Reply:
https://github.com/filecoin-project/storetheindex/pull/500